File: Modul3/C/PRAK304-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int bilangan;
    scanf("%d", &bilangan);

    if (bilangan >= 0 && bilangan < 100)
    {
        if (bilangan == 0)
        {
            printf("Nol\n");
        }
        else if (bilangan > 0 && bilangan < 10)
        {
            printf("Satuan\n");
        }
        else if (bilangan > 10 && bilangan < 20)
        {
            printf("Belasan\n");
        }
        else if (bilangan == 10 || bilangan >= 20)
        {
            printf("Puluhan\n");
        }
    }
    else
    {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    
    return 0;
}

File: Modul3/Python/PRAK304-2310817210003-MuhammadAzrianzan.py
bilangan = int(input(""))

if bilangan >= 0 and bilangan < 100:
    if bilangan == 0:
        print("Nol")
    elif bilangan > 0 and bilangan < 10:
        print("Satuan")
    elif bilangan > 10 and bilangan < 20:
        print("Belasan")
    elif bilangan == 10 or bilangan >= 20:
        print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")